SPECS/ctags/ctags.spec

```diff
@@ -25,6 +26,7 @@ TAGS files.
 %prep
 %setup -q
```

Comment:
While you're here, could you switch this to `%autosetup -p1`?

Reply:
I would love to do that, but one patch is `-p1` while the other is `-p2`;